tests/test-cases/large.rkt.out

```diff
@@ -283,11 +286,14 @@
        (define-syntax (id stx) (do-define-like stx #'internal-id)) ...
        (provide id ...))]))
 
-(provide-class-define-like-keyword [-field field] [-init init] [-init-field init-field])
+(provide-class-define-like-keyword [-field field]
+                                   [-init init]
+                                   [-init-field init-field])
 
 (define-for-syntax not-in-a-class
   (lambda (stx)
-    (if (eq? (syntax-local-context) 'expression)
+    (if (eq? (syntax-local-context)
+             'expression)
```

Comment:
This one isn't quite right. The `(eq? ...)` expression should still be on one line. Not sure why that happened.